packages/react/src/ErrorBoundary.tsx

```diff
@@ -187,7 +187,7 @@ export const ErrorBoundary = Object.assign(
   }
 )
 
-const ErrorBoundaryContext = createContext<({ reset: () => void } & ErrorBoundaryState) | null>(null)
+const ErrorBoundaryContext = createContext<Nullable<{ reset: () => void } & ErrorBoundaryState>>(null)
```

Comment:
In my opinion, this case is improvement than before. because we don't need to care parentheses to make type as nullable.